### PR TITLE
Takes skub to the next level.

### PR DESCRIPTION
**Pull request by SweptWasTaken**
Truly we have gone too far.

God bless bobbahbrown for [this](https://tgstation13.org/phpBB/viewtopic.php?f=5&t=26446&p=556480#p556480) thread on the tgstation forums and EOBGames who I bothered on discord to actually give me the sprite.

Discord_2020-05-26_11-46-27

### CODE REVIEW

**Comment by AJCM-git** [comment]
speedmerge, we need this

File: Resources/Prototypes/Entities/Items/Clothing/hats.yml [hunk 1]
```diff
@@ -2601,6 +2601,19 @@
...
+  name: Skub Hat
+  description: 'Huh, Skub.'
+  components:
+  - sprite: Clothing/Head/skubhead.rsi
+    type: Sprite
+  - sprite: Clothing/Head/skubhead.rsi
+    type: Icon
+  - sprite: Clothing/Head/skubhead.rsi
+    type: Clothing
+  parent: HatBase
```
Comment (AJCM-git) [inline Resources/Prototypes/Entities/Items/Clothing/hats.yml:2615]: put the parent above the id

Comment (SweptWasTaken) [reply Resources/Prototypes/Entities/Items/Clothing/hats.yml]: > 
> 
> put the parent above the id

Done but idk why you want this. Everything else in the yaml has a parents below the id...

File: Resources/Prototypes/Entities/Items/Clothing/hats.yml [hunk 2]
```diff
@@ -2601,6 +2601,19 @@
...
+  id: HatSkub
+  name: Skub Hat
+  description: 'Huh, Skub.'
+  components:
+  - sprite: Clothing/Head/skubhead.rsi
+    type: Sprite
+  - sprite: Clothing/Head/skubhead.rsi
+    type: Icon
+  - sprite: Clothing/Head/skubhead.rsi
+    type: Clothing
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Items/Clothing/hats.yml:2615]: You only had one job, Swept.

File: Resources/Prototypes/Entities/Items/Clothing/hats.yml [hunk 3]
```diff
@@ -2601,6 +2601,19 @@
...
+  id: HatSkub
+  name: Skub Hat
+  description: 'Best paired with the Skub Suit.'
+  components:
+    type: Sprite
+  - sprite: Clothing/Head/skubhead.rsi
+    type: Icon
+  - sprite: Clothing/Head/skubhead.rsi
+    type: Clothing
+  - sprite: Clothing/Head/skubhead.rsi
```
Comment (Zumorica) [inline Resources/Prototypes/Entities/Items/Clothing/hats.yml:2615]: ONE JOB.

Comment (Zumorica) [reply Resources/Prototypes/Entities/Items/Clothing/hats.yml:2615]:
```suggestion
  - type: Sprite
    sprite: Clothing/Head/skubhead.rsi
  - type: Icon
    sprite: Clothing/Head/skubhead.rsi
  - type: Clothing
    sprite: Clothing/Head/skubhead.rsi
```

**Comment by SweptWasTaken** [comment]
Epic

**Pull request merged**